apps/socketoptions.hpp

```diff
@@ -216,6 +216,7 @@ const SocketOption srt_options [] {
     { "inputbw", 0, SRTO_INPUTBW, SocketOption::POST, SocketOption::INT64, nullptr},
     { "oheadbw", 0, SRTO_OHEADBW, SocketOption::POST, SocketOption::INT, nullptr},
     { "latency", 0, SRTO_LATENCY, SocketOption::PRE, SocketOption::INT, nullptr},
+    { "tsbpd", 0, SRTO_TSBPDMODE, SocketOption::PRE, SocketOption::BOOL, nullptr},
```

Comment:
I know this is shorter and more handy, but the so far standard was that it's the lowercase name of the option without the `SRTO_` prefix...

Reply:
Do you mean `tsbpdmode` instead of `tsbpd`?